src/main/resources/templates/pollingunits.html

```diff
@@ -14,20 +14,60 @@
   <!-- Global site tag (gtag.js) - Google Analytics -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=UA-141725675-1"></script>
   <script type="text/javascript" th:src="@{/js/googleanalytics.js}"></script>
-  <!--Start of Tawk.to Script-->
-  <script type="text/javascript" th:src="@{/js/tawkto.js}"></script>
-  <!--End of Tawk.to Script-->
-  <!--JavaScript at end of body for optimized loading-->
-  <script type="text/javascript" th:src="@{/js/materialize.min.js}"></script>
-
 </head>
-<!-- Load our React component. -->
+<!--Start of Tawk.to Script-->
+<script type="text/javascript" th:src="@{/js/tawkto.js}"></script>
+<!--End of Tawk.to Script-->
 
 <body>
   <h2>Polling Units</h2>
-  <div id="react">
-    <script th:src="@{/built/bundle.js}"></script>
+  <div>
+    <table class="striped responsive-table">
+      <tr>
+        <th>#</th>
+        <th><a th:href="@{?sortBy=fullCode&sortDirection=ASC}">Code</a></th>
+        <th><a th:href="@{?sortBy=description&sortDirection=ASC}">Description</a></th>
+        <th><a th:href="@{?sortBy=longitude&sortDirection=ASC}">Longitude</a></th>
+        <th><a th:href="@{?sortBy=latitude&sortDirection=ASC}">Latitude</a></th>
+      </tr>
+      <tr th:each="pollingUnit, i: ${form.pollingUnitsPage.content}">
+        <td th:text="${(form.pollingUnitsPage.number*form.pollingUnitsPage.size)+i.index+1}"></td>
+        <td style="white-space: nowrap;">

```

Comment:
Putting styles in tags like this is highly discouraged. It might hunt you later

Reply:
Fixed in commit [a1e7951](a1e7951)